scripts/dist-management/upload-dist.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * Uploads a file to the webhintio/hint "dist" release.
+ * `node scripts/upload-dist.js fileName.zip`
+ *
+ * A valid GitHub auth token needs to be set in
+ * `process.env.GITHUB_TOKEN`
+ */
+
+const path = require('path');
+const fs = require('fs');
+
+const Octokit = require('@octokit/rest');
+const throttling = require('@octokit/plugin-throttling');
+
+const Client = Octokit.plugin(throttling);
+
+const octokitOptions = {
+    auth: `token ${process.env.GITHUB_TOKEN}`, // eslint-disable-line
```

Comment:
Nit: I'd prefer to only disable the specific `eslint` rule we want to suppress here (same goes for other `eslint-disable-line` instances).